spec/features/staff/beis_users_can_create_a_transfer_spec.rb

```diff
@@ -11,14 +11,19 @@
   end
 
   scenario "successfully creates a transfer" do
-    transfer = fill_in_transfer_form
+    quarter = FinancialQuarter.for_date(Date.today)
+
+    transfer = fill_in_transfer_form(
```

Comment:
Why are you overriding the quarter and year here, when your new defaults in `fill_in_transfer_form` will do the same thing..?

Reply:
Just to be a bit clearer when we're testing the outcome. Before, we were assuming that the Financial Year was Q1 2020, without explicity setting it. This way, we have access to an instance of the class too